### DISCUSSION TITLE
Getting "Please unblock challenges.cloudflare.com to proceed." in Cloudflare protected websites

### DISCUSSION THREAD

**Discussion by joaonunatings** [qutebrowser/qutebrowser#8071] [Q&A] [answered]
Title is self explanatory. Not sure if this is good enough for an issue, if it is, I can create it.
Screenshot of problem:
![image](https://github.com/qutebrowser/qutebrowser/assets/8835595/abce653b-2515-4c0c-97a0-27835ec7ab34)

**Comment by The-Compiler** [comment]
What's your `:version` info? Can you reproduce with `--temp-basedir`? If not, what about if you start with that but then do `:adblock-update`?

**Reply by joaonunatings** [reply, marked as answer]
> What's your `:version` info? 

https://bpa.st/DTQQ

---
I'm no longer having this issue so I couldn't reproduce it.
If I get it again I will try your procedures. I will close this for now.

EDIT: it might be from _Cloudflare_ side. my region is being _Re-routed_ ([status](https://www.cloudflarestatus.com/)).

**Reply by The-Compiler** [reply]
Might or might not be related, but you might want to upgrade to v3.1.0 which is based on Qt 6.6.

**Reply by joaonunatings** [reply]
Yes, will do, thank you :)
